session_items.py

```diff
@@ -67,3 +70,34 @@ def save_item(item):
     session['items'] = updated_items
 
     return item
+
+def update_item(id):
```

Comment:
Maybe a better name for this method is `complete_item`?

Reply:
Renamed